aspnetcore/tutorials/min-web-api.md

```diff
@@ -0,0 +1,348 @@
+
+## Install Postman to test the app
+
+This tutorial uses Postman to test the API.
+
+* Install [Postman](https://www.getpostman.com/downloads/)
+* Start the web app.
+* Start Postman.
+* Disable **SSL certificate verification**
+  * From **File** > **Settings** (**General** tab), disable **SSL certificate verification**.
+    > [!WARNING]
+    > Re-enable SSL certificate verification after testing the controller.
+
+<a name="post"></a>
+
+### Test PostTodoItem with Postman
+
+* Create a new request.
+* Set the HTTP method to `POST`.
+* Set the URI to `https://localhost:<port>/todoitems`. For example, `https://localhost:5001/todoitems`.
+* Select the **Body** tab.
+* Select the **raw** radio button.
+* Set the type to **JSON (application/json)**.
+* In the request body enter JSON for a to-do item:
+
+    ```json
+    {
+      "name":"walk dog",
+      "isComplete":true
+    }
+    ```
+
+* Select **Send**.
+
+  `![Postman with create request](min-web-api/_static/3/create.png)`
+
+## Examine the GET methods
+
+Two GET endpoints are implemented:
+
+* `GET /todoitems`
+* `GET /todoitems/{id}`
+
+Test the app by calling the two endpoints from a browser or Postman. For example:
+
+* `https://localhost:5001/todoitems`
+* `https://localhost:5001/todoitems/1`
+
+A response similar to the following is produced by the call to `Gettodoitems`:
+
+```json
+[
+  {
+    "id": 1,
+    "name": "Item1",
+    "isComplete": false
+  }
+]
+```
+
+### Test Get with Postman
+
+* Create a new request.
+* Set the HTTP method to **GET**.
+* Set the request URI to `https://localhost:<port>/todoitems`. For example, `https://localhost:5001/todoitems`.
+* Set **Two pane view** in Postman.
+* Select **Send**.
+
+This app uses an in-memory database. If the app is stopped and started, the preceding GET request will not return any data. If no data is returned, [POST](#post) data to the app.
+
+## Return values
+
+The return type of the `Gettodoitems` and `GetTodoItem` methods is `MinimalActionEndpointConventionBuilder`. ASP.NET Core automatically serializes the object to [JSON](https://www.json.org/) and writes the JSON into the body of the response message. The response code for this return type is [200 OK](https://developer.mozilla.org/docs/Web/HTTP/Status/200), assuming there are no unhandled exceptions. Unhandled exceptions are translated into 5xx errors.
+
+The return types can represent a wide range of HTTP status codes. For example, `GetTodoItem/{id}` can return two different status values:
+
+* If no item matches the requested ID, the method returns a [404 status](https://developer.mozilla.org/docs/Web/HTTP/Status/404) <xref:Microsoft.AspNetCore.Mvc.ControllerBase.NotFound%2A> error code.
+* Otherwise, the method returns 200 with a JSON response body. Returning `item` results in an HTTP 200 response.
+
+## Put methods
+
+Examine the `app.MapPut("/todoitems/{id}", ...` method:
+
+[!code-csharp[](min-web-api/samples/6.x/todo/Program.cs?name=snippet_put)]
```

Comment:
It doesn't look like `snippet_put` exists yet.

Reply:
That's why the title is "Double Draft", it's not up to draft status yet. :(